src/ceph-volume/ceph_volume/devices/lvm/activate.py

```diff
@@ -247,7 +247,10 @@ def activate_all(self, args):
                 )
             else:
                 terminal.info('Activating OSD ID %s FSID %s' % (osd_id, osd_fsid))
-                self.activate(args, osd_id=osd_id, osd_fsid=osd_fsid)
+                try:
+                    self.activate(args, osd_id=osd_id, osd_fsid=osd_fsid)
+                except Exception as e:
+                    terminal.error(f'Failed to activate: {e}')
```

Comment:
no one is going to look at the logs. do we want to change the exit status to error?

Reply:
May be.... But actually missed(failed) OSD to be reflected in cluster's status report anyway so user would probably care...
On the other hand error return might prevent some other stuff which calls ceph-volume from working and hence cause cluster's (or rather node's) outage.
So I'd rather prefer avoid error return for rather non-critical (from cluster's perspective) failures.